azure-pipelines.yml

```
@@ -39,6 +39,7 @@ steps:
     pip install -U qiskit jupyter sphinx nbsphinx sphinx_rtd_theme
     pip install 'matplotlib<3.3'
     pip install -U qiskit[visualization] cvxpy
+    pip install pyscf
```

Let see what CI says, but in other envs we've had to cap this as

```suggestion
    pip install 'pyscf<1.7.4'
```

because pyscf added a version cap on scipy which conflicts with what everything else installs